editor/src/components/navigator/navigator-item/navigator-item.tsx

```diff
@@ -545,6 +545,7 @@ export const NavigatorItem: React.FunctionComponent<
             selected={selected && !isInsideComponent}
             onMouseDown={collapse}
             style={{ transform: 'scale(0.6)', opacity: 'var(--paneHoverOpacity)' }}
+            testId={`navigator-item-collapse-${EP.toUid(props.navigatorEntry.elementPath)}`}
```

Comment:
I believe these will duplicate, there is a `navigatorEntryToKey` function IIRC which as long as we don't duplicate entries should generate IDs which are unique across the navigator.

Reply:
like this? https://github.com/concrete-utopia/utopia/pull/3485/commits/9385c64864250120e46655db7d49e8e9d4833e18

Reply:
Lovely!